tests/testthat/test-run-nmtran.R

```diff
@@ -0,0 +1,217 @@
+context("testing NM-TRAN")
+
+# define constants
+MODEL_DIR_BBI <- file.path(dirname(ABS_MODEL_DIR), "test-nmtran-models")
+NONMEM_DIR <- Sys.getenv("BBR_TESTS_NONMEM_DIR", "/opt/NONMEM")
+REQ_NONMEM_VERSIONS <- c("nm73gf", "nm74gf", "nm75", Sys.getenv("BBR_TESTS_NONMEM_VERSION"))
+REQ_NONMEM_VERSIONS <- unique(REQ_NONMEM_VERSIONS[REQ_NONMEM_VERSIONS != ""])
+
+# Don't assume NONMEM is available if not on Metworx.
+if (Sys.getenv("METWORX_VERSION") == "" || Sys.getenv("SKIP_BBI_TEST") == "true") {
+  skip("test-workflow-bbi only runs on Metworx because it needs NONMEM installed")
+}
+skip_long_tests("skipping long-running bbi workflow tests")
```

Comment:
These take only a second for on my end (once I drop the unnecessary sleep call on line 38).